providers/ms365/resources/ms365.lr

```diff
@@ -398,6 +398,16 @@ ms365.sharepointonline {
   spoTenant dict
   // SharePoint Online tenant sync client restriction
   spoTenantSyncClientRestriction dict
+  // SharePoint Online tenant sites
+  spoSites []ms365.sharepointonline.site
```

Comment:
@HRouhani i have made this to be a list, any reason why it cannot be?

Reply:
Thanks, looks really good.